site/content/docs/user-guide/using-custom-indexes.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Using Custom Plugin Indexes
+slug: custom-indexes
+weight: 800
+draft: true
+---
+
+Plugin indexes contain plugin manifests, which are documents that describe
+installation procedure for a plugin. For discovery purposes, Krew comes with a
+`default` plugin index, plugins hosted in the [`krew-index` repository][ki].
+
+However, some plugin authors may choose to host their own indexes that contain
+their curation of kubectl plugins. These are called "custom plugin indexes".
+
+## Adding a custom index
+
+A custom plugin index can be added with the `kubectl krew index add` command:
+
+```sh
+{{<prompt>}}kubectl krew index add foo https://github.com/foo/custom-index.git
+```
+
+The URI you use can be any [git remote](https://git-scm.com/docs/git-remote)
+(e.g., `git@github.com:foo/custom-index.git`).
+
+## Removing a custom index
+
+You can remove a custom plugin index by passing the name it was added with to
+the remove command:
+
+```sh
+{{<prompt>}}kubectl krew index remove foo
+```
+
+## Listing indexes
+
+To see what indexes you have added run the list command:
+
+```sh
+{{<prompt>}}kubectl krew index list
+{{<output>}}INDEX    URL
+default  https://github.com/kubernetes-sigs/krew-index.git
+foo      https://github.com/foo/custom-index.git{{</output>}}
+```
+
+## The default index
```

Comment:
This section should ideally come **after** installing plugins from custom indexes (keep in mind most people might not care about the default index nuances)